samples/batch_fetch_test.rb

```diff
@@ -11,7 +11,8 @@
 )
 
 batch_fetch_request = {
-  :filters => "BatchId=2956681"
+  :fields => "Files",
+  :filters => "BatchId=3009916"
```

Comment:
@zsalzbank - I was testing a different test batch. Where do you think they should be? You think the whole request object shouldn't be there? BatchFetch can also be called without any parameters, so it doesn't have to be there. I could just remove it?